applicationinsights.dev.json

```diff
@@ -7,7 +7,7 @@
   },
   "instrumentation": {
     "logging": {
-      "level": "DEBUG"
+      "level": "INFO"
```

Comment:
I've had a look at `DEBUG` as opposed to `INFO` and nothing really interesting at debug level